google/cloud/pubsub/quickstart/README.md

```diff
@@ -152,4 +164,4 @@ set GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=%cd%\roots.pem
 [grpc-roots-pem-bug]: https://github.com/grpc/grpc/issues/16571
 [homebrew-cmake-link]: https://formulae.brew.sh/formula/cmake
 [howto-setup-dev-workstation]: /doc/contributor/howto-guide-setup-development-workstation.md
-[pubsub-quickstart-link]: https://cloud.google.com/pubsub/docs/quickstart-console
+[pubsub-quickstart-link]: https://cloud.google.com/pubsub/docs/publish-receive-messages-client-library
```

Comment:
Note that moving the snippet tag from one file to another will require some matching change to the docs.

Reply:
What do you mean matching change to the docs?